spec/models/user_spec.rb

```diff
@@ -283,6 +283,19 @@
       expect(user.user_role_names).to include("Reviewer")
       expect(user.user_role_names.count).to eq(1)
     end
+
```

Comment:
@swatijadhav Can we add some end to end tests to prove that a user with an expired role cannot  access the API ? Maybe using timecop